### PR TITLE
Dynamic button

### PR DESCRIPTION
**Pull request by pwnsaur**

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -23,6 +23,12 @@ const nextConfig = {
     // },
     styledComponents: true,
   },
+
+  i18n: {
```
Comment (lluneva) [inline next.config.js:27]: hmm, tev būs lapa ar multi valodām ?

**Review by lluneva — APPROVED** [review]
Few comments, but nice

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import styled from 'styled-components';
+import SqareButton from '@/components/reusables/SquareButton';
```
[the remark this answers is not in the document]
Comment (pwnsaur) [reply src/components/Footer.tsx:2]: Paldies!

File: src/components/reusables/SquareButton.tsx [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  text-transform: uppercase;
+  font-weight: 600;
+  font-size: 1rem;
+  cursor: pointer;
+  transition: all 0.2s ease-in-out;
+
+  ${({ theme, $inverted }) => {
+    const { background, darkGreen } = theme.colors;
+
+    return css`
```
[the remark this answers is not in the document]
Comment (pwnsaur) [reply src/components/reusables/SquareButton.tsx:33]: thanks

**Pull request merged**